packages/clerk-js/src/ui/components/SignIn/SignInStart.tsx

```diff
@@ -64,7 +64,17 @@ export function _SignInStart(): JSX.Element {
     placeholder: localizationKeys('formFieldInputPlaceholder__password') as any,
   });
 
-  const identifierField = useFormControl('identifier', '', {
+  const initialValues: Record<SignInStartIdentifier, string | undefined> = useMemo(
+    () => ({
+      email_address: ctx.initialValues?.emailAddress,
+      email_address_username: ctx.initialValues?.emailAddress || ctx.initialValues?.username,
```

Comment:
❓ Can you elaborate on this key? Is it for an email or username case?

Reply:
Yes, this key is used for that specific case (where both of those are enabled in the instance).